Header header
geometry_msgs/Twist armorPose
geometry_msgs/Point[4] vertex
float64[9] translationCovariance
bool isBlue
bool isBig